### DISCUSSION TITLE
Use yt-dlp instead of youtube-dl

### DISCUSSION THREAD

**Discussion by Luffitys** [oliverjrose99/Recordurbate#62] [Ideas]
The reasoning behind my idea is the following: youtube-dl seems to not be updated any more and thus can cause troubles in the future. Yt-dlp is frequently being updated and has proven itself to be more reliable than youtube-dl in certain scenarios.
The repository of yt-dlp can be found here: https://github.com/yt-dlp/yt-dlp

Since yt-dlp requires either a manual installation via “pip” or an installation procedure utilizing “sudo wget” and “sudo chmod”, it would be necessary to refactor the Readme.md.

**Comment by Zeroth42** [comment]
How can you change the yt-dl to yt-dlp in the script and config?

**Comment by DatZig** [comment]
curious as well how to update the scripts to use yt-dlp instead of youtube-dl, anyone know what edits we need to make?

**Reply by divenxx** [reply]
in theory "youtube-dl_cmd" in the json config should do the trick but i havent tested it

**Reply by DatZig** [reply]
hmm .. yea worth a try ..  you have any idea how the logic of Recordurbate works in how it chooses who to record and how often?

**Reply by divenxx** [reply]
it is a thread running in the background checking for the list of configured models 
you start it with ./Recordurbate start
you add a model with ./Recordurbate add {Model_Name}

check this line https://github.com/oliverjrose99/Recordurbate/blob/3c4ea62fcad3d1ec6ab9bc350d84cbb9403102a6/recordurbate/bot.py#L124
